app/helpers/search_helper.rb

```diff
@@ -94,8 +94,7 @@ def project_select_tag
     options << [l(:label_my_projects), 'my_projects'] unless User.current.memberships.empty?
     options << [l(:label_and_its_subprojects, @project.name), 'subprojects'] unless @project.nil? || @project.descendants.active.empty?
     options << [@project.name, 'current_project'] unless @project.nil?
-    label_tag('scope', l(:description_project_scope), class: 'hidden-for-sighted') +
```

Comment:
Looks like you removed the label without replacement. At least impaired users will notice a difference.

Probably out of scope: To be honest, even non-impaired users might want to know that this select box is for the "Search Scope".